config/src/main/java/io/scalecube/config/ConfigRegistryImpl.java

```diff
@@ -81,25 +84,31 @@ protected boolean removeEldestEntry(Map.Entry<ConfigEvent, Object> eldest) {
         }
       };
 
+
   ConfigRegistryImpl(ConfigRegistrySettings settings) {
     Objects.requireNonNull(settings, "ConfigRegistrySettings can't be null");
     this.settings = settings;
   }
 
-  void init() {
+  public void init() {
     loadAndNotify();
-
-    reloadExecutor.scheduleAtFixedRate(
-        () -> {
-          try {
-            loadAndNotify();
-          } catch (Exception e) {
-            LOGGER.error("Exception on config reload, cause: {}", e, e);
-          }
-        },
-        settings.getReloadIntervalSec(),
-        settings.getReloadIntervalSec(),
-        TimeUnit.SECONDS);
+    if (this.schedule == null) {
```

Comment:
If the method has no public modifier we can remove the double checking stuff.

Reply:
it's a safety issue, we don't want to run several `configuration-load workers` (same as in https://github.com/scalecube/scalecube-config/pull/108#discussion_r284127541 )

Reply:
And we can't run them -> each will be created via `io.scalecube.config.ConfigRegistry#create` and only from this (we have no public constructor and the `init` method is package private as well) - so each invoking will create a new instance of ConfigRegistry.